src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = century
+    ? people.filter((x) =>
+      ((x.sex === 'm') && (century === Math.ceil(x.died / 100))))
+    : people.filter((x) => ((x.sex === 'm')));
+
+  const sumOfLifeYears = (sum, x) => sum + (x.died - x.born);
```

Comment:
Consider using a more descriptive name for the function 'sumOfLifeYears', such as 'sumAgeDifference'